src/test/bufferlist.cc

```diff
@@ -2600,8 +2600,8 @@ TEST(BufferList, ostream) {
   stream << bl;
   std::cerr << stream.str() << std::endl;
   EXPECT_GT(stream.str().size(), stream.str().find("list(len=6,"));
-  EXPECT_GT(stream.str().size(), stream.str().find("len 3 nref 1),\n"));
-  EXPECT_GT(stream.str().size(), stream.str().find("len 3 nref 1)\n"));
+  EXPECT_GT(stream.str().size(), stream.str().find("len 3 nref 1),"));
```

Comment:
@changchengx i don't understand this change. if you are not changing the output format, why bother changing the test?